hw2/04-background.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>04 - Background</title>
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+    />
+    <script defer src="04-background.js"></script>
+  </head>
+  <body class="bg-danger-subtle">
+    <main class="w-50 mx-auto border mt-5 p-3">
```

Comment:
Looking great! One thing I noticed, here you will want to change the background of the text container (like main class) or wrap in a div container and change the styling of that so that the background is `bg-light` boostrap class to match the example image.  I'm noticing wave accessibility errors because of the contrast issues.  aaaaand I just noticed it looks like this has been addressed already in someone's elses review. :) Should be an easy fix.